python/ray/test/test_modin.py

```diff
@@ -2,9 +2,22 @@
 from __future__ import division
 from __future__ import print_function
 
+import pytest
+
 import ray  # noqa F401
 
 
+def test_modin_import_with_ray_init():
```

Comment:
add a pytest fixture doing `ray.shutdown()` so that ray is shutdown properly, and pass it into all tests.